docs/prometheus/README.md

```diff
@@ -1,114 +1,129 @@
+it.
 
-This is a simple MongoDB resource with one user, and with the `spec.Prometheus`
-attribute with basic HTTP Auth and no TLS, that will allow you to test
-Prometheus metrics coming from MongoDB.
+The sample specifies a simple MongoDB resource with one user,
+and the `spec.Prometheus` attribute with basic HTTP 
+authentication and no TLS. The sample will allow you to test
+the Prometheus metrics coming from MongoDB.
 
 ## Quick Start
 
-We have tested this setup with version 0.54 of the [Prometheus
+We tested this setup with version 0.54 of the [Prometheus
 Operator](https://github.com/prometheus-operator/prometheus-operator).
 
-### Installing Prometheus Operator
+### Prerequisites
 
-The Prometheus Operator can be installed using Helm. Find the installation
-instructions
-[here](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack#kube-prometheus-stack):
+* Kubernetes 1.16+
+* Helm 3+
 
-This can be done with:
+### Install the Prometheus Operator
+
+You can install the Prometheus Operator using Helm. To learn 
+more, see the [installation instructions](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack#kube-prometheus-stack):
+
+To install the Prometheus Operator using Helm, run the 
+following commands:
 
 ``` shell
 helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
 helm repo update
-helm install prometheus prometheus-community/kube-prometheus-stack --namespace prometheus-system --create-namespace
+helm install prometheus prometheus-community/kube-prometheus-stack --namespace <prometheus-system> --create-namespace
 ```
 
-### Installing MongoDB
+### Install the MongoDB Community Kubernetes Operator
 
-*Change after release to a proper Helm install.*
-
-* Create a Namespace to hold our MongoDB Operator and Resources
+Run the following command to install the Community Kubernetes 
+Operator and create a namespace to contain the Community 
+Kubernetes Operator and resources:
 
 ``` shell
-kubectl create namespace mongodb
+helm install community-operator mongodb/community-operator --namespace <mongodb> --create-namespace
 ```
 
-* Follow the [Installation Instructions](https://github.com/mongodb/mongodb-kubernetes-operator/blob/master/docs/install-upgrade.md#operator-in-same-namespace-as-resources)
+To learn more, see the [Installation Instructions](https://github.com/mongodb/mongodb-kubernetes-operator/blob/master/docs/install-upgrade.md#operator-in-same-namespace-as-resources).
 
-## Creating a MongoDB Resource
+## Create a MongoDB Resource
 
-We have created a sample yaml definition that you can use to create a MongoDB
-resource and a `ServiceMonitor` that will indicate Prometheus to start scraping
-its metrics information.
+ You can use the [mongodb-prometheus-sample.yaml](https://github.com/mongodb/mongodb-kubernetes-operator/blob/master/docs/prometheus/mongodb-prometheus-sample.yaml) file to 
```

Comment:
**question**: it seems like this text is duplicated above. is this intended?

Reply:
yes